packages/tempus/src/Tempus_TimeStepControl_decl.hpp

```diff
@@ -117,6 +117,8 @@ class TimeStepControl
       { return tscPL_->get<int>   ("Maximum Order"); }
     virtual std::string getStepType() const
       { return tscPL_->get<std::string>("Integrator Step Type"); }
+    virtual bool getOutputExactly() const
```

Comment:
Did you mean to include Tempus changes?  It looks like something got mixed up with a rebase here.